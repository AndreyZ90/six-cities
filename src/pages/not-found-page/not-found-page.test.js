import React from 'react';
import TestRenderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';

import NotFoundPage from '@/pages/not-found-page/not-found-page';

describe(`NotFoundPage component snapshot`, () => {
  test(`Should correctly render NotFoundPage component`, () => {
    const tree = TestRenderer.create(
        <MemoryRouter>
          <NotFoundPage />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
