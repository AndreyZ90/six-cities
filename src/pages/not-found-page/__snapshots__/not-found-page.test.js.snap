// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component snapshot Should correctly render NotFoundPage component 1`] = `
<div
  className="page-not-found"
>
  <div
    className="page-not-found__container"
  >
    <p
      className="page-not-found__number"
    >
      404
    </p>
    <h1
      className="page-not-found__title"
    >
      Page not found :(
    </h1>
    <button
      className="page-not-found__button"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        go to the main page
      </a>
    </button>
  </div>
</div>
`;
